src/platform/unittest/TlsTest.cpp

```diff
@@ -96,6 +96,9 @@ struct TlsTest : public ::testing::TestWithParam<bool>
                 NULL,
                 QUIC_ALLOWED_CIPHER_SUITE_NONE};
             CredConfig.Flags |= CredFlags;
+#ifndef _WIN32
+            CredConfig.Flags |= QUIC_CREDENTIAL_FLAGS_USE_PORTABLE_CERTIFICATES;
+#endif
```

Comment:
Remove this now that we have a test explicitly for it?

Reply:
yes, I was meaning to but forget to do do it. updated.